FiveCalls/FiveCalls/Localizable.strings

```diff
@@ -132,12 +133,12 @@
 "subheading-message"                  = "You're making a difference! Calling is the most effective way to influence your representatives, and here's what you've achieved:";
 
 // welcome
-"welcome-page-1-title"                = "Make your voice heard";
-"welcome-page-1-message"              = "Turn your passive participation into active resistance. Facebook likes and Twitter retweets can’t create the change you want to see. Calling your Government on the phone can.";
-"welcome-page-2-title"                = "Spend 5 minutes, make 5 calls.";
-"welcome-page-2-message"              = "Calling is the most effective way to influence your representative.";
-"welcome-page-2-calls"                = "Together we've made\n%@ calls";
-"welcome-page-2-button-title"         = "Get Started";
+"welcome-section-1-title"                = "Make your voice heard";
+"welcome-section-1-message"              = "Turn your passive participation into active resistance. Facebook likes and Twitter retweets can’t create the change you want to see. Calling your Government on the phone can.";
+"welcome-section-2-title"                = "Spend 5 minutes, make 5 calls";
+"welcome-section-2-message"              = "Calling is the most effective way to influence your representative.";
+"welcome-section-3-calls"                = "Together we've made\n%@ calls";
+"welcome-button-title"                   = "Continue";
```

Comment:
I think "Get Started" makes more sense here than "Continue" since we only have one welcome screen and you get started right away